src/assets/img/homepage/angular-logo.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I think we already have an Angular logo asset already. Can that be reused?

Reply:
no, because that logo is angular white transparent logo